### PR TITLE
feature: adapt springcloud-alibaba-seata autoconfig

### PR DESCRIPTION
**Pull request by slievrly**
Signed-off-by: slievrly <slievrly@163.com>

### Ⅰ. Describe what this PR did

feature: adapt springcloud-alibaba-seata autoconfig

### Ⅳ. Describe how to verify it

image
image

### CODE REVIEW

File: seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    /**
+     * Gets tx service group.
+     *
+     * @return the tx service group
+     */
+    public String getTxServiceGroup() {
+        if (null == txServiceGroup) {
+            txServiceGroup = applicationId + DEFAULT_SPRING_CLOUD_SERVICE_GROUP_POSTFIX;
```
Comment (jsbxyyx) [inline seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: if user not configuration spring.application.name then applicationId is null?

Comment (xingfudeshi) [reply seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: > if user not configuration spring.application.name then applicationId is null?

If user does not configure this item,the default value is "application".
image

Comment (jsbxyyx) [reply seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: image
If user does not configuration, the applicationId is null, the txServiceGroup is null-seata-service-group?

Comment (xingfudeshi) [reply seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: > image
> If user does not configuration, the applicationId is null, the txServiceGroup is null-seata-service-group?

Yes,here could be null.

Comment (xingfudeshi) [reply seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: We can use "application" replace null.

Comment (jsbxyyx) [reply seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: I remember long long ago, the `spring.application.name` is must not be null.

File: seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java [hunk 2]
```diff
@@ -52,6 +58,10 @@ public String getApplicationId() {
      */
     public String getTxServiceGroup() {
         if (null == txServiceGroup) {
+            String applicationId = getApplicationId();
+            if (null == applicationId) {
+                LOGGER.warn(SPRING_APPLICATION_NAME_KEY + " is null, please set its value");
```
Comment (xingfudeshi) [inline seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: use {}

Comment (slievrly) [reply seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: fixed.

**Pull request merged**